_ap/0/index.adoc

```diff
@@ -0,0 +1,39 @@
+---
+num: 0
+title: Template 
+status: Draft
+authors: 
+  - "" # One item for each author, as github id or "firstname lastname"
+tags:
+  - "" # e.g. kafka, connectors, registry
+---
+
+// Top style tips:
+// * Use one sentence per line
+// * No unexpanded acronyms
+// * No undefined jargon
+
+// You don't have to use the following sections, but they provide a 
+// useful structure for writing a clear document.
+
+## Intent
```

Comment:
Reading these sections I have a harder time to project myself.
Can you provide a fictitious short example of an AP and apply the template?
This might even be part of the template to help less abstract people get onboard.

Another data point I'd like is some how who and how many services use that pattern to know when to decide to move to a PADR.

Reply:
> Can you provide a fictitious short example of an AP and apply the template?

@emmanuelbernard I've added an on-going example based on the metrics API which @JameelB is working on. Is that the sort of thing you were looking?

> Another data point I'd like is some how who and how many services use that pattern to know when to decide to move to a PADR.

That's not really something the AP itself is intended to address. IIRC we said that each time a service decided to follow an AP there should be a trivial ADR (with an `applies_ap` in the frontmatter) to record that fact ("Apply AP-X to Service Y"). But that in itself doesn't give us a number of times the AP has been used. Do you think we need that, or could we expect it to be caught during the review of the ADR?